Tests/format.test.ts

```diff
@@ -278,3 +287,21 @@ function vurderDelutfall(delutfall: Delutfall) {
     expect(delutfall.tekst).toBeDefined();
   }
 }
+
+
+
+/**
+ * Sjekker om et stegnummer er større enn et annet
+ * @param stegNr1 Stegnummer som skal sjekkes
+ * @param stegNr2 Stegnummer det skal sjekkes mot
+ * @returns 
+ */
+function stegnrErStorreEnn(stegNr1: string, stegNr2: string): boolean {
+  const [helTall1, des1] = stegNr1.split('.');
+  const [helTall2, des2] = stegNr2.split('.');
```

Comment:
For å være sikker på at stegnummerene er på riktig format, kan du finne nummerene med regexp i stedet: 

```
const value = "3.20";
const regex = /^(\d+)\.(\d+)$/;

const matches = value.match(regex);

if (matches) {
  const firstNumber = parseInt(matches[1]);
  const secondNumber = parseInt(matches[2]);
  ...
} else {
  // no match
  ...
}
```

`split` vil aldri feile, uansett hva input-stringen er.